kakarot_scripts/constants.py

```diff
@@ -223,7 +223,7 @@ class ChainId(IntEnum):
 ETH_TOKEN_ADDRESS = 0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7
 COINBASE = int(
```

Comment:
is this still used somewhere?

Reply:
it could be used in the deploy script when doing set_coinbase with a variable from environment